src/OpenTelemetry.Contrib.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta1

```

Comment:
```suggestion
## 1.0.0-beta1
```

Reply:
Oops. Looks like I had a newline in there markdownlint isn't happy. Mind committing the suggestion @cijothomas 
Thanks, sorry about that.

Reply:
i think you need to push a commit. (Commit suggestion is grayed out for me)